### PR TITLE
Added Disable Mouse Option

### PR DESCRIPTION
**Pull request by jamethy**
Added a disable mouse option so the scroll wheel could be used to switch workspaces. 

This option surely breaks a lot of modules, but I am using py3status only to add an external script and I still wanted to be able to switch workspaces with my mouse wheel.

### CODE REVIEW

File: py3status/cli.py [hunk 1]
```diff
@@ -101,6 +101,13 @@ def parse_cli():
         type=int,
         default=60,
         help="default injection cache timeout in seconds (default 60 sec)")
+    parser.add_argument(
+        '-m',
+        '--disable-mouse',
+        action="store_false",
+        dest="enable_mouse",
+        default=True,
+        help="disable all mouse events")
```
Comment (lasers) [inline py3status/cli.py]: Maybe just `disable mouse events`? Or @tobes, do you prefer broad `disable button clicks` instead?

**Comment by tobes** [comment]
maybe `click_events` like i3bar uses might be a better name.

@ultrabug do you have any thoughts?

**Comment by ultrabug** [comment]
Thanks a lot for your contribution @jamethy !

I agree with @tobes that I'd like to stay in line with i3bar naming. So could you please rename this feature to `disable_click_events`?

**Comment by jamethy** [comment]
This is my first ever oss contribution :smile: 

I ranmed it to `disable_click_events` and should be good to go

**Comment by ultrabug** [comment]
Thanks a lot for this @jamethy ! and congratulations for your first OSS contribution, I'm glad it was on py3status :)

**Comment by jamethy** [comment]
Hurray! :tada:

**Pull request merged**